schemas/dart_model.schema.json

```diff
@@ -0,0 +1,77 @@
+{
+  "$id": "https://github.com/dart-lang/macros/blob/main/schemas/dart_model.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "oneOf": [
+    {"$ref": "#/$defs/Model"}
+  ],
+  "$defs": {
+    "MetadataAnnotation": {
+      "type": "object",
+      "description": "A metadata annotation.",
+      "properties": {
+        "type": {"$ref": "#/$defs/QualifiedName"}
+      }
+    },
+    "Interface": {
+      "type": "object",
+      "description": "An interface.",
+      "properties": {
+        "metadataAnnotations": {
+          "type": "array",
+          "items": {"$ref": "#/$defs/MetadataAnnotation"}
+        },
+        "members": {
+          "type": "object",
+          "description": "Map of members by name.",
+          "additionalProperties": {"$ref": "#/$defs/Member"}
+        },
+        "properties": {"$ref": "#/$defs/Properties"}
+      }
+    },
+    "Member": {
+      "type": "object",
+      "description": "Member of a scope.",
+      "properties": {
+        "properties": {"$ref": "#/$defs/Properties"}
+      }
+    },
+    "Model": {
+      "type": "object",
+      "description": "Partial model of a corpus of Dart source code.",
+      "properties": {
+        "uris": {
```

Comment:
I don't understand what this property is for?

Reply:
Hmm actually maybe I did have a good reason: to avoid forcing that "things that are not libraries but go in files" end up in a different map, when probably they should go in the same map.

So maybe the Library type should be called File. URI -> file -> then semantics go on top of that.

Reply:
"things that are not libraries but go in files" What is an example of this?

Reply:
Part files, library augmentations, new types of thing that we dream up :)

I suggest we pull this plus "scopes" discussion to #10